.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
looks like this file should have been ignored from the `gitignore` above. can we remove it?